src/main/java/org/killbill/billing/catalog/api/PlanPhase.java

```diff
@@ -23,6 +23,8 @@
  */
 public interface PlanPhase extends CatalogEntity {
 
+    public StaticCatalog getCatalog();
```

Comment:
Should this be in `CatalogEntity` instead?

Reply:
I started that way as well but `CatalogEntity` i used by many other catalog resource that currently don't maintain a pointer to their static catalog.